greenhouse_api.gemspec

```diff
@@ -28,7 +28,10 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_dependency "faraday", '>= 0.15.4'
+  spec.add_dependency 'sorbet-runtime', '~> 0.5.5369'
+
   spec.add_development_dependency "rspec", "~> 3.10"
+  spec.add_development_dependency 'sorbet', '~> 0.5.5369'
```

Comment:
would love some feedback on this. this is the exact version our app uses, which is pretty much up to date. but it looks kinda weird.

Reply:
Let's use the following, assuming 0.5 is the latest stable. This gives users flexibility to upgrade sorbet independently

```ruby
spec.add_development_dependency 'sorbet', '>= 0.5'
```